static/app/views/onboarding/setupDocs.tsx

```diff
@@ -66,13 +65,6 @@ function SetupDocs({location, recentCreatedProject: project}: StepProps) {
                   projectSlug={project.slug}
                   platform={currentPlatform.name}
                 />
-              ) : params.showLoader ? (
```

Comment:
We can now remove `showLoader` from `useOnboardingQueryParams`, right?

Reply:
thank you! yes it is done here https://github.com/getsentry/sentry/pull/77699